apps/settings/test/unit/panels/root/stk_item_test.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+/* globals MockNavigatorMozIccManager, MockDump,
+  MocksHelper, MockNavigatorSettings */
+requireApp('settings/shared/test/unit/load_body_html_helper.js');
+requireApp('settings/shared/test/unit/mocks/mock_navigator_moz_icc_manager.js');
+requireApp('settings/shared/test/unit/mocks/mock_dump.js');
+require('/shared/test/unit/mocks_helper.js');
+
+var mocksForIccMenu = ['Settings'];
+
+mocksForIccMenu.forEach(function(mockName) {
+  if (!window[mockName]) {
+    window[mockName] = null;
+  }
+});
+
+suite('STK Item >', function() {
+  var modules = [
+    'panels/root/stk_item',
+    'unit/mock_settings',
+    'shared_mocks/mock_l10n'
+  ];
+
+  var map = {
+    '*': {
+      'modules/settings': 'unit/mock_settings',
+      'shared/l10n' : 'shared_mocks/mock_l10n'
+    }
```

Comment:
same here, make sure we would map `shared/stk_helper` to `shared_mocks/mock_stk_helper`